floodviz/views.py

```diff
@@ -18,8 +17,32 @@ def home():
 
     j = hydrograph_utils.req_hydrodata(sites, start_date, end_date, url_top)
     all_series_data = hydrograph_utils.parse_hydrodata(j)
-    
-    with open('floodviz/static/data/hydrograph_data.json', 'w') as fout: 
+
+    with open('floodviz/static/data/hydrograph_data.json', 'w') as fout:
         json.dump(all_series_data, fout, indent=1)
 
     return render_template('index.html')
+
+
+@app.route('/map/')
+def sitemap():
+    site_data = map_utils.site_dict(app.config['SITE_IDS'], app.config['NWIS_SITE_SERVICE_ENDPOINT'])
+    site_data = map_utils.create_geojson(site_data)
+    projection = map_utils.projection_info(app.config['PROJECTION_EPSG_CODE'], app.config['SPATIAL_REFERENCE_ENDPOINT'])
+
+    # TODO: the following is a hack:
```

Comment:
Where did the counties.json file come from? Right now I believe we only need to draw state boundaries but it would be good to clarify with the Read and/or Walker. It does not belong in the instance directory at any rate. So  that explains the comment.